Website/readme.md

```diff
@@ -1 +1,5 @@
-This website is used to help display the Student Council's duty list in a organised and neat way.
\ No newline at end of file
+This website is used to help display the Student Council's duty list in a organised and neat way.
+
+**Versions:**
+- Bootstrap: **v5.0.2**
```

Comment:
Codacy has a fix for the issue: [Lists should be surrounded by blank lines](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8389313)
```suggestion

- Bootstrap: **v5.0.2**
```